src/Lib/View.php

```diff
@@ -154,6 +154,14 @@ final public function fetch($file = null, $view = null)
             extract($this->vars);
         }
 
+        // In some instances when running minPHP on macOS from a network drive
+        // or an external drive through a symbolic link, the file path is not
+        // correct due to the fact that "$this->view_path" may already include ROOTWEBDIR
+        if (!file_exists($file) && strpos(strtolower(PHP_OS), 'darwin') !== false) {
```

Comment:
Is this meant to be `PHP_OS` or `PHP_OS_FAMILY`?

Also, this work around looks to treat `::view_path` differently than intended by the rest of the class, particularly with how `::setView` (called above) sets the `::view_path` for use via `::getViewPath`. Perhaps a change to `::getViewPath` and how it determines the correct view path to return would be a better area to update for this particular use case? `::getViewPath` already removes ROOTWEBDIR when the view has exactly two parts, so (depending on this use case) it could do something similar.

In any case, it would be ideal to see some tests added to be sure this update doesn't introduce any regressions. `.travis.yml` should also [define operating systems to test against](https://docs.travis-ci.com/user/multi-os/) since your update makes a distinction depending on the underlying operating system.